### PR TITLE
[DOCS] Adding Conan distribution article to docs for master

### PR DESCRIPTION
**Pull request by msmykx-intel**
Added article about Conan Package Manager to the documentation.

Jira ticket: 105004

### CODE REVIEW

File: docs/articles_en/get started/installing-openvino-overview/installing-openvino-shared/installing-openvino-conan.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+@sphinxdirective
+
+.. meta::
+   :description: Learn how to install OpenVINO™ Runtime on Windows, Linux, and 
+                 macOS operating systems, using Conan Package Manager.
+
+.. note::
+   
+   Note that the Conan Package Manage distribution:
```
Comment (kblaszczak-intel) [inline docs/articles_en/get started/installing-openvino-overview/installing-openvino-shared/installing-openvino-conan.md:11]:
```suggestion
   Note that the Conan Package Manager distribution:
```

File: docs/articles_en/get started/installing-openvino-overview/installing-openvino-shared/installing-openvino-conan.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+   * is dedicated to users of all OSs (Windows, Linux, macOS, Solaris, FreeBSD, etc.)
```
Comment (kblaszczak-intel) [inline docs/articles_en/get started/installing-openvino-overview/installing-openvino-shared/installing-openvino-conan.md:13]:
```suggestion
   * is dedicated to users of all major OSs: Windows, Linux, macOS
```

File: docs/articles_en/get started/installing-openvino-overview/installing-openvino-shared/installing-openvino-conan.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+   * it can be used for any existing platfrom: desktop, mobile, embbeded, servers, bare metal, etc.
```
Comment (kblaszczak-intel) [inline docs/articles_en/get started/installing-openvino-overview/installing-openvino-shared/installing-openvino-conan.md:14]:
```suggestion

```

File: docs/articles_en/get started/installing-openvino-overview/installing-openvino-shared/installing-openvino-conan.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+   * can manage any compiler in any of their versions.
+   * OpenVINO™ is included in the Conan Center Index - registry with the software packages.
```
Comment (kblaszczak-intel) [inline docs/articles_en/get started/installing-openvino-overview/installing-openvino-shared/installing-openvino-conan.md:16]:
```suggestion
  
```

**Comment by ilya-lavrenov** [comment]
looks like you have not included Conan to "Distribution comparison for OpenVINO 2023.1" table https://docs.openvino.ai/2023.1/openvino_docs_install_guides_overview.html?VERSION=v_2023_1_0&OP_SYSTEM=WINDOWS&DISTRIBUTION=ARCHIVE

Conan provides the same devices as vcpkg

**Pull request merged**